### ISSUE TITLE
Marker Previous Command Incorrect

### ISSUE THREAD

**Issue by waynchi** [VSpaceCode/VSpaceCode#23]
The command to see the previous error is incorrectly called as:
"editor.action.marker.previous"

The correct command is:
"editor.action.marker.prev"

```
    {
        "before": [
            "<leader>",
            "e",
            "p"
        ],
        "after": [],
        "commands": [
            {
                "command": "editor.action.marker.prev",
                "args": []
            }
        ]
    }
```
![editor command](https://user-images.githubusercontent.com/3287927/52598651-4ba2f700-2e0b-11e9-8d98-6e4ccb989377.png)

**Comment by StreakyCobra** [comment]
Hi @waynchi,

Thanks for spotting this, I will have a look at this later this week :)

**Comment by StreakyCobra** [comment]
Fixed in a29902e7b8df27ad47a641660447bb1a261d219e
